tests/integration/components/tasks-test.js

```diff
@@ -12,7 +12,7 @@ module('Integration | Component | tasks', function (hooks) {
 
```

Comment:
Do we need more tests to cover the new feature(s)?

Reply:
yes, we definitely need tests, there are no tests as of now, since this feature had a tight deadline

Reply:
@swarajpure Then let's please have a ticket that tracks the follow-up of tests with timeline/ETA